packages/webpack-cli/package.json

```diff
@@ -41,6 +41,7 @@
     "colorette": "^2.0.14",
     "commander": "^10.0.1",
     "cross-spawn": "^7.0.3",
+    "dotenv-webpack-plugin": "1.0.0",
```

Comment:
Move it to `peerDependencyMeta` and mark it as a peer dependency, just like `webpack-bundle-analyzer`.

Reply:
We anyway have to publish it as a package on npm, otherwise even inside `dependency` it won't be installed. The comment just suggests we keep the code of the plugin in webpack-cli repo only so that we can maintain it easily.

Reply:
Why we need this plugin?

Reply:
@alexander-akait This is our new package inside `webpack-cli/packages/dotenv-webpack-plugin`

Reply:
done